internal/seleniumtest/seleniumtest.go

```diff
@@ -1098,6 +1100,50 @@ func testGetAttributeNotFound(t *testing.T, c Config) {
 	}
 }
 
+func testGetProperty(t *testing.T, c Config) {
+	wd := newRemote(t, newTestCapabilities(t, c), c)
+	defer quitRemote(t, wd)
+
+	if err := wd.Get(c.ServerURL); err != nil {
+		t.Fatalf("wd.Get(%q) returned error: %v", c.ServerURL, err)
+	}
+	input, err := wd.FindElement(selenium.ByName, "q")
+	if err != nil {
+		t.Fatalf("Can't find element: %v", err)
+	}
+	const query = "golang"
+	if err := input.SendKeys(query); err != nil {
+		t.Fatalf("Can't send keys: %v", err)
+	}
+
+	time.Sleep(500 * time.Millisecond)
+
+	val, err := input.GetProperty("no-such-property")
```

Comment:
If this property doesn't exist, how does this test pass?

Reply:
Ah, I see: the following commit fixes it. While I'm going to squash these commits when the PR is approved for this particular PR, I recommend in general squashing commits in advance in such cases, especially if tests don't pass on intermediate commits.

Reply:
I will do it next time. (I usually use the "Files changed" tab and there the commits do not matter.)